commit 7097ed1c2580d4b02b84d78d1173ecaf82794365
Author: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date:   Wed Oct 31 14:59:40 2018 +0000

    MAINTAINERS: Add entry for BCM2835 Unicam driver
    
    Adds entry for the new BCM2835 Unicam (CSI-2 receiver) driver
    
    Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>

diff --git a/MAINTAINERS b/MAINTAINERS
index 9e9b19ecf6f7..7c9d9ebcaf7a 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -2873,6 +2873,13 @@ S:	Maintained
 N:	bcm2835
 F:	drivers/staging/vc04_services
 
+BROADCOM BCM2835 CAMERA DRIVER
+M:	Dave Stevenson <dave.stevenson@raspberrypi.org>
+L:	linux-media@vger.kernel.org
+S:	Maintained
+F:	drivers/media/platform/bcm2835/
+F:	Documentation/devicetree/bindings/media/bcm2835-unicam.txt
+
 BROADCOM BCM47XX MIPS ARCHITECTURE
 M:	Hauke Mehrtens <hauke@hauke-m.de>
 M:	Rafał Miłecki <zajec5@gmail.com>
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1302_7097ed1c2580d4b02b84d78d1173ecaf82794365.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1302_7097ed1c2580d4b02b84d78d1173ecaf82794365.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
